gtk/DetailsDialog.cc

```diff
@@ -994,6 +996,40 @@ void DetailsDialog::Impl::refreshInfo(std::vector<tr_torrent*> const& torrents)
 
     dl_lb_->set_text(str);
 
+    // dl_ts_lb
+    if (stats.empty())
+    {
+        str = no_torrent;
+    }
+    else
+    {
+        auto const downloaded_str = tr_strlsize(std::accumulate(
+            std::begin(stats),
+            std::end(stats),
+            uint64_t{ 0 },
+            [](auto sum, auto const* st) { return sum + st->downloadedThisSession; }));
+
+        auto const failed = std::accumulate(
+            std::begin(stats),
+            std::end(stats),
+            uint64_t{ 0 },
+            [](auto sum, auto const* st) { return sum + st->corruptThisSession; });
+
+        if (failed != 0)
+        {
+            str = fmt::format(
+                _("{downloaded_size} (+{discarded_size} discarded after failed checksum)"),
```

Comment:
A bit unsure about the `+`. Does downloaded size include corrupted size (I guess so since corruption is detected after a piece is downloaded)? If so, adding one value to another (which is suggested by the `+`) makes no sense. Maybe just drop it?

Reply:
This is using the same text as the Qt client, which was added in #2605 for #154.

I feel like the `+` is useful for 154. The value on the left is actually how much data has been downloaded + passed the checksum test.

Would `"{downloaded_size} verified (+{discarded_size} discarded after failed checksum)"` be better? worse?